### PR TITLE
2.0.4 release

### PR DESCRIPTION
**Pull request by finnlewis**

### CODE REVIEW

**Comment by finnlewis** [comment]
Tests now running on this, after fixing merge requests from changes to test.yml

**Comment by finnlewis** [comment]
@stephen-cox the [tests that are still failing on this ](https://github.com/localgovdrupal/localgov_project/actions/runs/3128455156/jobs/5235994018#step:5:20)are I think the tests that you mentioned, the functional JavaScript tests that seem to work fine locally, but not on the GitHub CI

So do we just ignore them for now and get this released?

**Comment by finnlewis** [comment]
Just to confirm, all tests do pass locally: 

```
www-data@59c3c77ede62:/app$ phpunit
PHPUnit 9.5.25 #StandWithUkraine

Testing 
...............................................................  63 / 150 ( 42%)
............................................................... 126 / 150 ( 84%)
........................                                        150 / 150 (100%)

Time: 29:40.852, Memory: 42.00 MB

OK (150 tests, 3235 assertions)

File: .github/workflows/test.yml [hunk 1]
```diff
@@ -7,6 +10,12 @@ on:
   pull_request:
     branches:
       - '2.x'
+  schedule:
+    - cron: "0 7 * * *"
```
Comment (Adnan-cds) [inline .github/workflows/test.yml:14]: Now we will have one failed test run every morning :)

**Review by Adnan-cds — APPROVED** [review]
Thanks :)

**Pull request merged**